tools/engine_tool/lib/main.dart

```diff
@@ -65,6 +65,11 @@ void main(List<String> args) async {
     configs: configs,
   );
 
-  io.exitCode = await runner.run(args);
+  try {
+    io.exitCode = await runner.run(args);
+  } on FatalError catch (e, st) {
+    print(st);
```

Comment:
This might race with the logger's write to io.stderr. They could be printed to the terminal in either order. If there's a problem with writing to `io.stderr`, the `Logger` will fall back to `print`, so using the same `Logger` that is handed to the environment should be safe here.

Reply:
Replaced with a Logger.error call.